lesson_05/main.py

```diff
@@ -18,16 +18,19 @@ def main():
     print("rock paper scissors!")
     user_input = None
 
-    while user_input != "q":
+    while user_input != quit_game:
         print("choose your weapon: Rock, Paper, Scissors or Q to quit:")
         user_input = input()
         pc_input = random.choice(rock_paper_scissors)
-        print(f"{user_input} vs {pc_input}")
+        if user_input == quit_game:
+            print("bye")
+        elif user_input != pc_input or user_input == pc_input:
+            print(f"{user_input} vs {pc_input}")
 
-        if user_input == "q":
+        if user_input == quit_game:
             break
         elif user_input == pc_input:
-            tie()
+            report_winner("tie")
```

Comment:
We wanted to send report_winner something else when there's a tie.

Try to remember what we talked about... What do we conceptually want to send when there no winner (as opposed to a winner whose name is "no one" or "tie")?